index.html

```diff
@@ -169,6 +169,33 @@ <h3>Sponsors & Partners</h3>
     </div>
   </section>
 
+  <section id="rules-and-faq" class="wrapper style2 fade-up">
+    <div class="inner">
+      <h2>Rules</h2>
+      <p>The CODEVID-19 Hackathon is divided into two phases. Weekly prizes are awarded until April 30th, when projects can submit their projects for the final prize consideration. Following judging final prizes are awarded.</p>
```

Comment:
> Following judging final prizes are awarded.

Can be tweaked:

> Final prizes are awarded following a judging period.

Do we have an idea of how long we should be giving judges to do so?  I'm feeling like a week is good.  We should add this in once we know what the time scale will be.

Reply:
I'll udpated this next week once we've talked to judges and prepared judging resources. Until then I'm not comfortable making a specific commitment publicly. Privately, we've talked about aiming to award all prizes within a week.

Reply:
I've changed to the verbage you suggest.